app/code/Apruve/Payment/Observer/ShipmentObserver.php

```diff
@@ -242,56 +244,6 @@ protected function _getDiscountItem($object)
         }
     }
 
-    protected function _apruve($action, $token, $data = '', $object = 'orders', $requestType = 'POST')
```

Comment:
Why was this deleted? I though this was how we did network calls?

Reply:
We use _processShipment now, because this call requires two completely different Nouns than _apruve was built to use.